parsing/dialects/intermediate/dialects/dialect1/src/test/java/org/apache/calcite/test/Dialect1ParserTest.java

```diff
@@ -4232,6 +4232,24 @@ private void checkExNotNull(SqlParserUtil.StringAndPos sap, Throwable thrown) {
     sql(sql).ok(expected);
   }
 
+  @Test public void testRenameProcedureTo() {
```

Comment:
Are these tests meant to be part of this PR?

Reply:
I think they got rearranged when I merged master, I can move them back down so it doesn't say it's added